pages/1.11/installing/faq/index.md

```diff
@@ -3,17 +3,16 @@ layout: layout.pug
 navigationTitle:  Frequently Asked Questions
 title: Frequently Asked Questions
 menuWeight: 203
-excerpt: Frequently asked questions about installing DC/OS Enterprise
+excerpt: Frequently asked questions about installing DC/OS 
 
-enterprise: false
 ---
 
 
 ## Q. Can I install DC/OS on an already running Mesos cluster?
 We recommend starting with a fresh cluster to ensure all defaults are set to expected values. This prevents unexpected conditions related to mismatched versions and configurations.
 
 ## Q. What are the OS requirements of DC/OS?
-See the [system requirements](/1.11/installing/ent/custom/system-requirements/).
+See the [system requirements](/1.11/installing/production/system-requirements/).
```

Comment:
Broken link, check again before merging

Reply:
@judithpatudith I'll do a test on each redirect once I finish the grouping.